deploy-dev.sh

```diff
@@ -1,2 +1 @@
-docker build . -t scbd/online-reporting-tool && docker push scbd/online-reporting-tool && \
-ssh ubuntu@cbddev.xyz 'sudo docker pull scbd/online-reporting-tool && sudo docker service update --force frontend-2_nr7'
\ No newline at end of file
+ssh ubuntu@cbddev.xyz 'sudo docker pull scbd/online-reporting-tool:dev && sudo docker service update --force frontend-2_ort'
```

Comment:
If you want to have the build on the script, you can build from mac arm fro interl using:
```docker buildx build --platform linux/amd64 ...``` 
instead of 
```docker build ...```